API/things-api/app.js

```diff
@@ -170,6 +203,28 @@ app.get('/stats/range/:start_date/:end_date', function(err, client, done) {
 });
 
 
+
+/****************************************************
+* /path     /stats/range/:start_date/:end_date
+* /params   :start_date - Beginning of time frame
+*           :end_date - End of time frame
+*
+* /brief    Responds with a meaningful status and returns the results
+*
+* /author   Luke
+* /date     2/7/2017
+****************************************************/
+var errResultHandler = function(err, result, res) {
```

Comment:
I recommend commenting and grouping this and the transaction functions for clarity.